genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -364,6 +364,10 @@ number of CPU cores x 2 + 1
 to the number of CPU cores x 2 + 1
 |1
 
+|genie.tasks.userMetrics.enabled
```

Comment:
might want to make this key `user-metrics` now to save yourself problems in master with the changes to property bindings spring did

Reply:
Just kept going with convention used by other similar switches..

Reply:
Those all had to be flipped if they were camel case in 4.x. If you're ok with making sure you do that then sure leave it like this. Just pointing it out that you could just do it once for a cleaner cherry-pick if you're going for that.